tests/k8s.sh

```diff
@@ -16,7 +16,7 @@ set -ex
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 TEST_DIR=$(mktemp -d)
-CLADM=$SCRIPT_DIR/../bin/cl-adm
+CLI=$SCRIPT_DIR/../bin/cl
```

Comment:
should be `clusterlink`? I think this is what caused the accidental typo at the top of file
```suggestion
CLI=$SCRIPT_DIR/../bin/clusterlink
```